Unexpected failure of `docker/login-action` reproducibility workflow

The [`docker-login-action-v3.yml`](https://github.com/ericcornelissen/reproducing-actions/blob/aec4d2ac6f23efa1847077ebc1be8524a69668fa/.github/workflows/docker-login-action-v3.yml) workflow is failing unexpectedly due to usage of the wrong version of `yarn`.

The error log (taken from [`docker/login-action@v3 #91`](https://github.com/ericcornelissen/reproducing-actions/actions/runs/9311848020/job/25631675802)):

```log
$ Run yarn install
error This project's package.json defines "packageManager": "yarn@3.6.3". However the current global version of Yarn is 1.22.22.

Presence of the "packageManager" field indicates that the project is meant to be used with Corepack, a tool included by default with all official Node.js distributions starting from 16.9 and 14.19.
Corepack must currently be enabled by running corepack enable in your terminal. For more information, check out https://yarnpkg.com/corepack.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/docker-login-action-v3.yml
+++ b/.github/workflows/docker-login-action-v3.yml
@@ -23,7 +23,9 @@ jobs:
     with:
       files:        dist/index.js
       build-cmd:    yarn build
-      install-cmd:  yarn install
+      install-cmd:  |
+        corepack enable
+        yarn install
       node-version: 20.x
       repository:   docker/login-action
       version-ref:  v3

